app/src/main/res/xml/status_widget_info.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:description="@string/widget_description"
+    android:initialKeyguardLayout="@layout/status_widget"
+    android:initialLayout="@layout/status_widget"
+    android:minWidth="@dimen/widget_min_size"
+    android:minHeight="@dimen/widget_min_size"
+    android:previewImage="@mipmap/ic_launcher"
+    android:previewLayout="@layout/status_widget"
+    android:resizeMode="none"
+    android:targetCellWidth="1"
+    android:targetCellHeight="1"
+    android:updatePeriodMillis="1800000" />
```

Comment:
Don't remember if I asked about it before, but how was this decided?

Reply:
`1800000` here (30 minutes) is the minimum supported (by Android) auto-update interval.
`0` (zero) can be specified here to disable automatic updates.
I think, auto-updates can be disabled, since the widget is updated when `ACTION_STARTED` or `ACTION_STOPPED` are received.

Reply:
Is it updated when you start or stop from inside the app? If yes, then we should probably disable auto updates. If not, maybe we actually increase the interval. Maybe to a couple of hours.

Reply:
The widget is notified when the service is started/stopped. I will disable auto-update with `android:updatePeriodMillis="0"`.